src/azul/health.py

```diff
@@ -151,3 +159,46 @@ def _lambda(self, lambda_name) -> JSON:
             return {
                 'up': up,
             }
+
+
+class HealthController(Health):
```

Comment:
Controllers shouldn't inherit models because the [is-a](https://en.wikipedia.org/wiki/Is-a) relationship does not apply to [controllers and models](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller). The easy counter example is that one controller often need to deal with multiple model instances and then the subtype relation breaks down. Also, in typical web apps the controllers are singletons that outlive the models.